src/router.ts

```diff
@@ -39,6 +39,7 @@ export const router = createRouter({
       path: '/action',
       component: HandleAction,
       props: (route: RouteLocation) => ({
+        continueUrl: route.query.continueUrl,
```

Comment:
is this different from redirect_uri? if not, lets keep the name consistent

Reply:
The query param must be named continueUrl: https://firebase.google.com/docs/auth/custom-email-handler#create_the_email_action_handler_page (see the params table)